includes/html/functions.inc.php

```diff
@@ -1678,3 +1678,154 @@ function get_unit_for_sensor_class($class)
 
     return $units_by_classes[$class];
 }
+
+/**
+ * Rounds down to the closest number divisible by 5.
+ */
+function lowest_five($number)
+{
+    return floor($number / 5) * 5;
+}
+
+/**
+ * Rounds down to the nearest 5 minutes if using seconds.
+ */
+function lowest_five_minutes($time)
```

Comment:
This function could be made more generic adding the interval as a second optional argument defaulting to 300